tests/test_helper/bats/services.bats

```diff
@@ -0,0 +1,80 @@
+
+load "${ABS_TOP_TEST_DIRNAME}test_helper/setup_teardown/$(basename "${BATS_TEST_FILENAME//.bats/.bash}")"
+
+setup() {
+    load ${ABS_TOP_TEST_DIRNAME}test_helper/common.bash
+    load ${ABS_TOP_TEST_DIRNAME}test_helper/lxd.bash
+    load ${ABS_TOP_TEST_DIRNAME}test_helper/microovn.bash
+    load ${ABS_TOP_TEST_DIRNAME}../.bats/bats-support/load.bash
+    load ${ABS_TOP_TEST_DIRNAME}../.bats/bats-assert/load.bash
+
+    # Ensure TEST_CONTAINERS is populated, otherwise the tests below will
+    # provide false positive results.
+    assert [ -n "$TEST_CONTAINERS" ]
+}
+
+services_register_test_functions() {
```

Comment:
These tests seem to be heavily dependent on order of execution which is not the best practice. If forced order of operations is really necessary, it should be a single test. Otherwise each test should restore state of the system on teardown.

As it may be a bit hard to restore system state on failure, I'd perhaps suggest to go with single long test here that bails in case of any failure.